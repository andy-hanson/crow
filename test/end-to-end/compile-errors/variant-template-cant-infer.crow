main void()
	info log "{7::nat.x f}"
	()

v variant mut

x[t] record(value t) v variant-member

f nat(a v)
	match a
	as x value
		value
	else
		0
