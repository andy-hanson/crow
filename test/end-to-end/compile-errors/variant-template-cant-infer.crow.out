exit code: 1
stderr:
test/end-to-end/compile-errors/variant-template-cant-infer.crow 11:8-11:9 Can't infer type arguments of x
Uncaught exception: Reached compile error
	at f
	at main
